Integrated the global header and footer in leaderboard page

Prabaldas-wd opened the pull request.

PeterOche commented:
your application is not showing on onlydust. can you make sure to apply from the app ?
also you're meant to remove the buttons at the bottom (Hoeme and Play Game)

Prabaldas-wd commented:
> you're meant to also remove the buttons at the bottom (Hoeme and Play Game) @Prabaldas-wd

Hi @PeterOche , i have removed the buttons(Hoeme and Play Game) at the bottom and updated the PR. Can you please check ?

PeterOche commented:
> > you're meant to also remove the buttons at the bottom (Hoeme and Play Game) @Prabaldas-wd
> 
> Hi @PeterOche , i have removed the buttons(Hoeme and Play Game) at the bottom and updated the PR. Can you please check ?

please remove unused imports too

Prabaldas-wd commented:
> please remove unused imports too, after removing the buttons

Hi @PeterOche, I have updated the PR by removing the unused imports.

PeterOche reviewed (approved):
lgtm. thank you for your contribution

Prabaldas-wd commented:
> lgtm. thank you for your contribution

Thanks for the opportunity!
